src/riak_kv_ts_api.erl

```diff
@@ -32,6 +32,7 @@
          get_data/2, get_data/3, get_data/4,
          delete_data/2, delete_data/3, delete_data/4, delete_data/5,
          query/2,
+         row_to_key/3,
```

Comment:
No, it's different. And it's a simple convenience thing: https://github.com/basho/riak_kv/pull/1404/files#diff-4f95cc846bf31c44a5533591faee0ef6L181.